pkg/parser/parser.peg

```diff
@@ -964,6 +967,7 @@ DoubleQuoteBoldTextElement <- Word
         / MonospaceText
         / SubscriptText
         / SuperscriptText
+        / InlineIcon
```

Comment:
could you add a few tests to verify that inline icons/images can be parsed in quoted text?

Reply:
Yes.